distributed/tests/test_utils.py

```diff
@@ -59,7 +59,8 @@
 )
 
 
-def test_All(loop):
+@gen_test()
```

Comment:
`@gen_test` should take care of providing a pristine IOLoop because it invokes `test_All` within a `clean()` context?

Reply:
Yep